src/intTest/java/uk/nhs/digital/nhsconnect/nhais/controller/RemovalIntegrationTest.java

```diff
@@ -0,0 +1,131 @@
+package uk.nhs.digital.nhsconnect.nhais.controller;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.io.IOException;
+import java.nio.file.Files;
+
+import lombok.extern.slf4j.Slf4j;
+import uk.nhs.digital.nhsconnect.nhais.IntegrationTestsExtension;
+import uk.nhs.digital.nhsconnect.nhais.model.fhir.ParameterNames;
+import uk.nhs.digital.nhsconnect.nhais.model.fhir.ParametersExtension;
+import uk.nhs.digital.nhsconnect.nhais.parse.FhirParser;
+
+import org.hl7.fhir.r4.model.OperationOutcome;
+import org.hl7.fhir.r4.model.Parameters;
+import org.hl7.fhir.r4.model.Patient;
+import org.hl7.fhir.r4.model.StringType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.io.Resource;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+@ExtendWith({ SpringExtension.class, IntegrationTestsExtension.class})
+@SpringBootTest
+@AutoConfigureMockMvc
+@Slf4j
+public class RemovalIntegrationTest {
```

Comment:
What's the purpose of this test? I see it's using `outbound_uat_data/removal/app-j-1.fhir.json` which will be picked up by UAT tests automatically

Reply:
it uses the file, but then modifies some of the parameters to provide not happy path cases

Reply:
it is done similar way that it has been done for Deduction

Reply:
👍